website/src/app/api.service.ts

```diff
@@ -172,6 +172,14 @@ export class ApiService {
     );
   }
 
+  closeIssue(issueID: string): Observable<Issue> {
+    return this.http.put<Issue>(`/api/v1/issues/${issueID}/closeIssue`, undefined, {
+    }).pipe(
+      tap(data => console.log("closing issue", data)),
+      catchError(this.handleError<Issue>("closeIssue", undefined)),
```

Comment:
We ought to make sure that if we are unable to close the issue, we give a popup with an error so the user knows it wasn't closed.